app/config/FrontendAppConfig.scala

```diff
@@ -37,6 +37,7 @@ class FrontendAppConfig @Inject() (configuration: Configuration,  servicesConfig
   val betaFeedbackUnauthenticatedUrl = s"$contactHost/contact/beta-feedback-unauthenticated"
   val signOutUrl: String             = configuration.get[String]("urls.logout")
 
+  val discloseArrangeLink: String = configuration.get[String]("urls.homepage")
```

Comment:
Do we need this? This is the same as urls.loginContinue.

Reply:
@daniel-manning suggested having a separate declaration from urls.loginContinue.